app/assets/stylesheets/modules/modal.css.scss

```diff
@@ -31,7 +31,7 @@
   overflow-y: scroll;
 
   ul {
-    list-style: circle;
+    list-style: none;
```

Comment:
Are you sure you want to make this change here? Won't this affect all modal lists? In some cases we will probably want these markers.

Reply:
No - I am not sure about any css changes. It is up to you and @jsiarto. But I know that the dot is used only in cwic 'learn more' modal.

Reply:
Ok. So @jsiarto, when you start styling these related url links, just make sure that this `list-style` rule is in a place that doesn't bork other things.